### ISSUE TITLE
Unexpected need for score corrections at STG Wannsee event

### ISSUE THREAD

**Issue by aksajhfduwafe** [aksajhfduwafe/bugzilla-to-github-issues-test#1045]
| Field | Value |
|---|---|
| **Reporter** | @jens-rommel |
| **Assignee** | @JakobTracTrac |
| **Product** | TracTrac System |
| **Component** | APIs |
| **Version** | unspecified |
| **Hardware** | All |
| **OS** | All |
| **Severity** | critical |
| **Priority** | P1 |
| **Created** | 2012-11-14T16:22:25Z |
| **Original Bug** | [Bug 1045](https://bugzilla.sapsailing.com/bugzilla/show_bug.cgi?id=1045) |

---

At the Wannsee event http://obmr2012.sapsailing.com there was an unexpected high need for score corrections, even in cases when the finish was clear.

E.g.
- Leaderboard "Vorrunde", race V5 blue (TracTrac "OBMR 2012 VR Sa-Vm Fleet 1 2"):
-- 470er Männer: without score correction is ranked 5th, actually finished 3rd

**Comment by aksajhfduwafe** [comment]
**@jens-rommel** commented:

Created attachment 96
Official race results as texted from Mark Bayer

[official_results.txt](https://github.com/aksajhfduwafe/bugzilla-to-github-issues-test-attachments/raw/main/1045/96_official_results.txt) — Official race results as texted from Mark Bayer (text/plain, 2022 bytes)

**Comment by aksajhfduwafe** [comment]
**@axeluhl** commented:

Did you already try re-loading the races from TracTrac to see if the finish line passing are still not matching the official scores?

**Comment by aksajhfduwafe** [comment]
**@axeluhl** commented:

Today, we ran a test with the Extreme Sailing Series 2012 Rio act data, based on races R15-R18. We started the tracking for all four races. R16 started with an empty course. While R15 ran, Jakob updated the R16 course which worked fine. He also modified the course of R15 twice while the race was one which also worked fine. After R15 had finished, we left our tracker running, and R16, as expected, produced start line mark passings for R16. This time, contrary to what we experienced in Rio, the R15 mark passings remained unaffected as we had hoped. Jakob also tried creating replay data of previous races that we were still tracking, and this also created no problems at all. Resolving as FIXED.
